### PR TITLE
add Asset and Person examples, add state to resources

### PR DESCRIPTION
**Pull request by michzimny**

### CODE REVIEW

File: docs/asset.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "type": "Property",
+        "value": "Red hammer no. 58"
+    },
+    "status": {
+        "type": "Property",
+        "value": "available",
+        "observedAt": "2020-12-01T11:23:19Z"
+    },
+    "@context": [
+        "https://smartdatamodels.org/context.jsonld"
```
Comment (matteopantano) [inline docs/asset.md]: Should we not add also the context of shop4cf? `https://raw.githubusercontent.com/shop4cf/data-models/master/docs/shop4cfcontext.jsonld`

File: docs/asset.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+It may be a tool or an element of a device.
+
+For instance, a gripper for a robot (i.e. for a device) or a hammer for a person are assets.
+
+Example of a hammer is given below.
+
+```json
+{
+    "id": "urn:ngsi-ld:Asset:company-xyz:hammer-xqi7wnx",
+    "type": "Asset",
```
Comment (matteopantano) [inline docs/asset.md:14]: We are missing the `isSpecifiedBy` we discussed last time together. The example we said was as follows: 
```
"isSpecifiedBy": {
        "type": "Relationship",
        "object": "urn:ngsi-ld:ResourceSpecification:company-xyz:BPMNid"
 }
```

File: docs/person.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Person
+
+Person is a human (human worker) who can perform a step in manufacturing.
+
+```json
+{
+    "id": "urn:ngsi-ld:Person:company-xyz:worker-xqi7wnx",
+    "type": "Person",
+    "firstName": {
```
Comment (matteopantano) [inline docs/person.md:9]: `isSpecifiedBy` is missing as said above

File: docs/person.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        "type": "Property",
+        "value": "Smith"
+    },
+    "status": {
+        "type": "Property",
+        "value": "available",
+        "observedAt": "2020-12-01T11:23:19Z"
+    },
+    "@context": [
+        "https://smartdatamodels.org/context.jsonld"
```
Comment (matteopantano) [inline docs/person.md]: Context of shop4cf is missing

**Review by matteopantano — COMMENTED** [review]
Files are reviewed

File: docs/asset.md (hunk 2 above)
Comment (michzimny) [reply docs/asset.md:14]: Yes, but I'm only wondering whether `isSpecifiedBy` shouldn't be able to point at multiple Resource Specifications. Let's discuss (re-discuss?) this.

File: docs/asset.md (hunk 1 above)
Comment (michzimny) [reply docs/asset.md]: Yes, but the following issue has to be resolved first.

Attribute `status` in Task has different meaning than in Resources. In Task, it is defined as in #5. And in all Resources:
> The concrete structure of the `status` attribute depends on a specific use case.

So we are unable to give a single definition of `status` in that context. 

Thus, should we rename attribute `status` in all resources to something else, e.g. `resourceStatus`? And then separately define `status` and `resourceStatus` in the context?

Comment (kontrag) [reply docs/asset.md]: Indeed, task status and resource status have different values, so I agree with Michal that we should/could give different names.

Comment (michzimny) [reply docs/asset.md]: We discussed and agreed on changing to `state`.

File: docs/asset.md (hunk 2 above)
Comment (michzimny) [reply docs/asset.md:14]: We discussed and agreed that this relation is many-to-many.

This change will be more complex, so I would prepare to address it in another PR.

File: docs/person.md (hunk 4 above)
Comment (michzimny) [reply docs/person.md]: fixed

File: docs/asset.md (hunk 1 above)
Comment (michzimny) [reply docs/asset.md]: fixed

**Comment by michzimny** [comment]
@matteopantano @kontrag If you do not have further comments, I would merge this PR, and create another one to add `isSpecifiedBy` to all resources.

**Comment by matteopantano** [comment]
Good for me like this

**Pull request merged**